src/orange/BoundingBoxUtils.hh

```diff
@@ -267,14 +267,14 @@ inline U calc_dist_to_inside(BoundingBox<T> const& bbox,
                 continue;
             }
 
-            U dist = (bbox.point(static_cast<Bound>(bound))[ax] - pos[ax])
-                     / dir[ax];
+            T temp = (bbox.point(static_cast<Bound>(bound))[ax] - pos[ax]);
 
-            if (dist < 0)
+            if (temp * signum(dir[ax]) <= 0)
```

Comment:
Oh yeah I forgot that the sign of the direction mattered 🤦 it's more readable and possibly faster the original way you wrote it, sorry!